---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

You can also find my articles on 
<a href="https://scholar.google.com/citations?user=wc5LHGcAAAAJ">my Google Scholar profile</a>.

*  **Mikhail Kolmogorov**, Mikhail Rayko, Jeffrey Yuan, Evgeny Polevikov & Pavel Pevzner.
[metaFlye: scalable and accurate long-read metagenome assembler](https://www.biorxiv.org/content/10.1101/637637v1), _bioRxiv_ (2019)

* Evgeny Polevikov and **Mikhail Kolmogorov**. 
[Synteny paths for assembly graphs comparison](http://drops.dagstuhl.de/opus/volltexte/2019/11054/), _WABI_ (2019)

* **Mikhail Kolmogorov**, Jeffrey Yuan, Yu Lin & Pavel Pevzner. 
[Assembly of long error-prone reads using repeat graphs](https://www.nature.com/articles/s41587-019-0072-8),
_Nature Biotechnology_ 37, 540–546 (2019)

* Alla Mikheenko & **Mikhail Kolmogorov**.
[Assembly Graph Browser: interactive visualization of assembly graphs](https://academic.oup.com/bioinformatics/article-abstract/35/18/3476/5306331),
_Bioinformatics_ 35 (18), 3476-3478 (2019)
			
* **Mikhail Kolmogorov**, Joel Armstrong, Brian J. Raney, Ian Streeter, Matthew Dunn, Fengtang Yang, Duncan Odom, Paul Flicek, Thomas Keane, David Thybert, Benedict Paten & Son Pham.
[Chromosome assembly of large and complex genomes using multiple references](https://genome.cshlp.org/content/28/11/1720.full),
_Genome Research_ 28 (11), 1720-1732 (2018)

* Alex Bishara, Eli L Moss, **Mikhail Kolmogorov**, Alma Parada, Ziming Weng, Arend Sidow, Anne E Dekas, Serafim Batzoglou, Ami S Bhatt.
[High-quality genome sequences of uncultured microbes by assembly of read clouds](https://www.nature.com/articles/nbt.4266),
_Nature Biotechnology_ 36, 1067–1075 (2018)

* Jingtao Lilue, Anthony G Doran, Ian T Fiddes, Monica Abrudan, Joel Armstrong, Ruth Bennett, William Chow, Joanna Collins, Stephan Collins, Anne Czechanski, Petr Danecek, Mark Diekhans, Dirk-Dominik Dolle, Matt Dunn, Richard Durbin, Dent Earl, Anne Ferguson-Smith, Paul Flicek, Jonathan Flint, Adam Frankish, Beiyuan Fu, Mark Gerstein, James Gilbert, Leo Goodstadt, Jennifer Harrow, Kerstin Howe, Ximena Ibarra-Soria, **Mikhail Kolmogorov**, Chris J Lelliott, Darren W Logan, Jane Loveland, Clayton E Mathews, Richard Mott, Paul Muir, Stefanie Nachtweide, Fabio CP Navarro, Duncan T Odom, Naomi Park, Sarah Pelan, Son K Pham, Mike Quail, Laura Reinholdt, Lars Romoth, Lesley Shirley, Cristina Sisu, Marcela Sjoberg-Herrera, Mario Stanke, Charles Steward, Mark Thomas, Glen Threadgold, David Thybert, James Torrance, Kim Wong, Jonathan Wood, Binnaz Yalcin, Fengtang Yang, David J Adams, Benedict Paten, Thomas M Keane,
[Sixteen diverse laboratory mouse reference genomes define strain-specific haplotypes and novel functional loci](https://www.nature.com/articles/s41588-018-0223-8),
_Nature Genetics_ 50, 1574–1583 (2018)

* David Thybert, Maša Roller, Fábio CP Navarro, Ian Fiddes, Ian Streeter, Christine Feig, David Martin-Galvez, **Mikhail Kolmogorov**, Václav Janoušek, Wasiu Akanni, Bronwen Aken, Sarah Aldridge, Varshith Chakrapani, William Chow, Laura Clarke, Carla Cummins, Anthony Doran, Matthew Dunn, Leo Goodstadt, Kerstin Howe, Matthew Howell, Ambre-Aurore Josselin, Robert C Karn, Christina M Laukaitis, Lilue Jingtao, Fergal Martin, Matthieu Muffato, Stefanie Nachtweide, Michael A Quail, Cristina Sisu, Mario Stanke, Klara Stefflova, Cock Van Oosterhout, Frederic Veyrunes, Ben Ward, Fengtang Yang, Golbahar Yazdanifar, Amonida Zadissa, David J Adams, Alvis Brazma, Mark Gerstein, Benedict Paten, Son Pham, Thomas M Keane, Duncan T Odom, Paul Flicek,
[Repeat associated mechanisms of genome evolution and function revealed by the Mus caroli and Mus pahari genomes](https://genome.cshlp.org/content/28/4/448.short),
_Genome Research_ 28 (4), 448-459 (2018)

* **Mikhail Kolmogorov**, Eamonn Kennedy, Zhuxin Dong, Gregory Timp and Pavel Pevzner,
[Single-Molecule Protein Identification by Sub-Nanopore Sensors](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005356)
_PloS Computational Biology_ 13 (5), e1005356 (2017)

* Yu Lin*, Jeffrey Yuan*, __Mikhail Kolmogorov*__, Max Shen and Pavel Pevzner,
[Assembly of long error-prone reads using de Bruijn graphs](https://www.pnas.org/content/113/52/E8396.short)
PNAS 113 (52), E8396-E8405, (2016), * equal contribution

* **Mikhail Kolmogorov**, Xiaowen Liu and Pavel Pevzner,
[SpectroGene: a tool for proteogenomic annotations using top-down spectra](https://pubs.acs.org/doi/abs/10.1021/acs.jproteome.5b00610),
_Journal of Proteome Research_ 15 (1), 144-151 (2015)

* **Mikhail Kolmogorov**, Brian Raney, Benedict Paten, Son Pham,
[Ragout - a reference-assisted assembly tool for bacterial genomes](https://academic.oup.com/bioinformatics/article-abstract/30/12/i302/388572),
_Bioinformatics_ 30 (12), i302-i309 (2014)

* Ilya Minkin, Anand Patel, **Mikhail Kolmogorov**, Nikolay Vyahhi, Son Pham,
[Sibelia: a scalable and comprehensive synteny block generation tool for closely related microbial genomes](https://link.springer.com/chapter/10.1007/978-3-642-40453-5_17),
WABI (2013)
